physionet-django/user/forms.py

```diff
@@ -298,6 +299,7 @@ class RegistrationForm(forms.ModelForm):
                     widget=forms.TextInput(attrs={'class': 'form-control'}),
                     validators=[validate_name])
 
+    privacy_policy = forms.BooleanField(required=True, label=mark_safe(settings.PRIVACY_POLICY_HTML))
```

Comment:
Could we wrap this in a `if privacy_policy` clause, so that platforms have the ability to switch the feature off?